nautobot/core/templates/admin/index.html

```diff
@@ -2,14 +2,14 @@
 {% load i18n static bootstrapped_goodies_tags %}
 
 {% block breadcrumbs %}
-<ul class="breadcrumb">
+<ol class="breadcrumb">
 <li>{% trans 'Admin Home' %}</li>
-</ul>
+</ol>
 {% endblock %}
 
 
 {% block object-tools %}
-<ul class="nav navbar-nav">
+<ol class="nav navbar-nav">
```

Comment:
Unlike breadcrumbs, the navbar examples in bootstrap are using `ul` rather than `ol`, and that matches what we have in `nautobot/core/templates/inc/nav_menu.html`, so let's keep this as `ul` for now.

Reply:
Oops, I don't think I meant to change that.  I will remove this change.